packages/google_maps_flutter/google_maps_flutter_ios/CHANGELOG.md

```diff
@@ -12,132 +16,132 @@
 
 ## 2.8.1
 
-* Improves Objective-C type handling.
+- Improves Objective-C type handling.
```

Comment:
These style changes don't seem to be related to your change. Please follow existing style in the changelog, consistent with the others.